catalog/opensearch/catalog-opensearch-source/src/test/java/org/codice/ddf/opensearch/source/OpenSearchFilterVisitorTest.java

```diff
@@ -942,21 +989,19 @@ public void testMultipleSpatialFilters() {
                 .intersecting()
                 .wkt(anotherWktPolygon);
 
-    final And andFilter =
-        geotoolsFilterBuilder.allOf(
+    final Or orFilter =
+        geotoolsFilterBuilder.anyOf(
```

Comment:
❓Do the failure messages in the assertions against `pointRadiusSearches` need to be updated? They still reference using an AND filter. (there may be others messages that also need to be updated)